web/server.js

```diff
@@ -1,7 +1,6 @@
 // modules =================================================
 var express		= require("express");
 var app			= express();
-var mongoose	   = require("mongoose");
```

Comment:
Lots of stuff got deleted and then inserted. In this case, was "mongoose" purposely removed? @DylanWalseth